scripts/init.sh

```diff
@@ -36,8 +39,13 @@ echo "Installing web packages"
 echo "Download geth to replace the nix version"
 OS=$(uname -s | tr A-Z a-z)
 MACHINE_TYPE=$(uname -m | tr A-Z a-z)
+if [ "$OS" == "linux" ]; then
+  MACHINE_TYPE="amd64"
+fi
```

Comment:
This should work for mac 64 bit intel as well with the following modification:
```suggestion
MACHINE_TYPE=$(uname -m | tr A-Z a-z | sed 's/x86_64/amd64/')
```